src/components/FlowRunGraphPopover.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div ref="target" class="flow-run-graph-popover" :style="styles">
+    <slot />
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { computed, onMounted, onUnmounted, ref } from 'vue'
+
+  const props = defineProps<{
+    position?: {
+      x: number,
+      y: number,
+    },
+  }>()
+
+  const emit = defineEmits<{
+    'onClose': [null],
+  }>()
+
+  const target = ref<HTMLElement | null>(null)
+
+  const styles = computed(() => ({
+    top: `${props.position?.y ?? 0}px`,
+    left: `${props.position?.x ?? 0}px`,
+  }))
+
+  onMounted(() => {
+    setTimeout(() => {
+      document.addEventListener('click', eventHandler)
+      document.addEventListener('focusin', eventHandler)
+    }, 0)
+  })
+
+  onUnmounted(() => {
+    document.removeEventListener('click', eventHandler)
+    document.removeEventListener('focusin', eventHandler)
+  })
```

Comment:
You could use [`useGlobalEventListener`](https://github.com/prefecthq/vue-compositions/tree/main/src/useGlobalEventListener) for this so you don't have to manager the lifecycle

Reply:
I do need my `setTimeout` so it doesn't immediately call `eventHandler`. Curious your take on this, do I...
A. leave it as is.
B. Update useGlobalEventListener to have an option for like `immediate = false`
C. I could manage a flag that's set after the tick and checked in the eventHandler function